lib/main.coffee

```diff
@@ -33,9 +35,21 @@ module.exports =
   createRecentFilesView: ->
     unless @recentFilesView?
       RecentFilesView = require './recent-files-view'
-      @recentFilesView = new RecentFilesView(@recentFiles)
+      @recentFilesView = new RecentFilesView(@getMetricsReporter(), @recentFiles)
     @recentFilesView
 
+  getMetricsReporter: ->
```

Comment:
Considering we do not care about these metrics, and do not want to pollute stats for FuzzyFinder, I'd prefer don't even use their implementation. Instead, I suggest we create a proxy object that makes any function calls no-ops.

```suggestion
  getMetricsReporter: ->
    new Proxy(new Function, {
      get: (target, prop, receiver) -> receiver,
    })
```